### ISSUE TITLE
Improvements to installation and usage documentation

### ISSUE THREAD

**Issue by bignose-debian** [adrianoveiga/django-media-fixtures#5]
The documentation for install and usage can be improved. I will make some clarifications and propose a merge request.

**Comment by bignose-debian** [comment]
At my personal fork is a branch ‘[maintenance/documentation](https://git.sr.ht/~bignose/django-media-fixtures/tree/maintenance/documentation)’ that changes the README document with some improvements.

This is currently based on ‘master’ and should merge cleanly.

**Comment by adrianoveiga** [comment]
Hi, @bignose-debian , thank you for improving the README file!
I checked this link of your repository and "diff" them by myself. It's pretty good.
But can you create a Pull Request here on Github? Otherwise, I will copy the code and push by myself, ok?

**Comment by bignose-debian** [comment]
On 25-Aug-2020, adrianoveiga wrote:
> Hi, @bignose-debian , thank you for improving the README file!
> I checked this link of your repository and "diff" them by myself.
> It's pretty good.

Thanks!

Oh, you don't need to diff them manually. Git can do this easily, when
you add my repository as another remote (temporarily, if you want to).

Then you can compare branches and merge and all the other useful Git
operations.

> But can you create a Pull Request here on Github?

No, I don't keep any content on GitHub; the great thing about a
distributed VCS like Git is we can work on different providers and
still collaborate with full Git features.

> Otherwise, I will copy the code and push by myself, ok?

Sure. You can add my repository as a remote (with whatever name you
like, e.g. ‘bignose’) and fetch its branches; then, compare my feature
branch to your local ‘master’, and merge if you want to.

-- 
 \     “I know you believe you understood what you think I said, but I |
  `\         am not sure you realize that what you heard is not what I |
_o__)                                     meant.” —Robert J. McCloskey |
Ben Finney <ben@benfinney.id.au>

**Comment by adrianoveiga** [comment]
Thank you again for the answer!

When I said "diff them by myself" I expressed badly. I didn't do it manually, but copied the file and used a diff program. But I did not think to add your repository as another remote... ha ha! :-)

This is why I asked if I can "copy the code and push", because this way the commit would be only in my name as if I made it alone.
But adding your repo as another remote and merge it solves everything and is the correct solution! Thank you!

I was thinking the most lazy solution, to speak the truth, because I was not on my computer with git to get the code, to commit and push. With a Pull Request, I could easily merge with one button in Github. ha ha ha :-D
